lib/playlistGenerator/PlaylistGenerator.js

```diff
@@ -66,9 +65,10 @@ function PlaylistGenerator (entryObject, options) {
     that.persistPromise = qResolved;
 
 
-
     if (_.has(options, 'recording') && options.recording == true){
+        that.loggerInfo = `[${entryObject.entryId}-${entryObject.recordedEntryId}]`;
         that.playListFormat = 'recording';
+        that.recordedEntryId = entryObject.recordedEntryId;
```

Comment:
doesn't sound logic that the playlistgenerator should keep recordedEntryId

Reply:
These changes were done to enable better debugging and a way to separate log messages related to live stream from recording in context of playlistGenerator. removed the recordingEntryId as class member, left the loggerInfo